src/front/DataInspector.js

```diff
@@ -2,7 +2,18 @@ L.TileLayer.XRay = L.TileLayer.extend({
 
     getTileUrl: function (tilePoint) {
         this.options.version = Date.now();
-        this.options.showLayer = L.K.Config.dataInspectorLayer || '';
+        var keys = Object.keys(L.K.Config);
+        if (L.K.Config['dataInspectorLayer___all__'] === true) {
```

Comment:
You need to make sure `L.K.Config.dataInspector` exists and is an object before calling FormBuilder. FormBuilder expects an existing object.

I need to add a bit of documentation do Leaflet.FormBuilder :/ I'll do that with the new version of uMap, as soon as Leaflet 1.0 is finally released.